src/fcm/index.js

```diff
@@ -2,35 +2,77 @@ const crypto = require('crypto');
 const request = require('request-promise');
```

Comment:
Nit: i don't think we need `request-promise` any longer. Shall we drop this at the same time?

Reply:
Yeah can remove

Reply:
Ah it's actually used [here](https://github.com/superhuman/push-receiver/pull/10/files#diff-7b823abc5c68e6f7480ff869f94ab2db9e425ac74f557f37925d36cef3aef3e5R24) going to leave as is